maat-court-data-api/src/main/java/gov/uk/courtdata/util/DateUtil.java

```diff
@@ -4,6 +4,10 @@
 
 public final class DateUtil {
 
+    private DateUtil() {
```

Comment:
Could use constructor annotation here.

Reply:
As discussed, This is private Cons, Dont want to over engineer it with combinations of annotations